source/manual/create-a-local-transaction.html.md

```diff
@@ -0,0 +1,236 @@
+
+This can be configured with or without `button_text` and `button_link`. [Seek content design advice](#seek-content-design-advice) with the content.
+
+For example:
+
+```yaml
+LGSL_CODE:
+  unavailable_in:
+    Scotland:
+      title: "This service is not available in Scotland"
+      body: "We do not know if they offer this service. Search the Scottish Government website to see if they do, or what else they offer."
+      button_text: "Find information for Scotland"
+      button_link: "https://www.mygov.scot"
+    Northern Ireland:
+      title: "This service is not available in Northern Ireland"
+      body: "This service is not available in Northern Ireland. You can find other services on your council website"
+```
+
+## Add the service to Local Links Manager
+
+__Note__: These steps must be complete before starting:
+
+* [Get details of or create a service](#get-details-of-or-create-a-service)
+* [Add the service to Publisher](#add-the-service-to-publisher)
+
+Run [this rake task](https://github.com/alphagov/local-links-manager/blob/master/lib/tasks/service.rake) in LLM to create (if necessary), hook up and enable the service...
+
+```bash
+bundle exec rails service:enable[LGSL_CODE,LGIL_CODE,SERVICE_NAME,SERVICE_SLUG]
+```
+
+Then run [this rake task](https://github.com/alphagov/local-links-manager/blob/master/lib/tasks/import/missing_links.rake) in Local Links Manager to add (empty) links for all available local authorities...
+
+```bash
+bundle exec rails import:missing_links
+```
+
+## Upload Local Authority URLs to Local Links Manager
+
+__Note__: These steps must be complete before starting:
+
+* [Get details of or create a service](#get-details-of-or-create-a-service)
+* [Get a list of local authority slugs and URLs](#get-a-list-of-local-authority-slugs-and-urls)
+* [Add the service to Local Links Manager](#add-the-service-to-local-links-manager)
+
+Log into Backend and upload the completed CSV file...
+
+```bash
+$ gds govuk connect ssh -e integration backend
+
+# If you run this rake task in Jenkins, you'll need run scp-push against all
+# backend machines. Otherwise you can omit `:MACHINE_NO`.
+$ gds govuk connect scp-push -e integration backend:MACHINE_NO LOCAL_FILE_PATH REMOTE_FILE_PATH
+```
+
+Run [this rake task](https://github.com/alphagov/local-links-manager/blob/master/lib/tasks/import/service_links_from_csv.rake) to import the links from the CSV file.
+
+```bash
+bundle exec rails import:service_links[LGSL_CODE,LGIL_CODE,REMOTE_FILE_PATH]
+```
+
+## Add content and publish the service in Publisher
+
+Once __all__ of the above steps are complete, ask a member of your content design team to add any desired content and publish the service in [Publisher](https://publisher.publishing.service.gov.uk).
```

Comment:
Maybe add that they will need the LGSL and LGIL code

Reply:
Also that they need to create the page as a new 'Local transaction' format in publisher

Reply:
Thanks @hannahjump 👍 updated